tutorials/llm/llama-3/pruning-distillation/05_display_results.ipynb

```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6c91263b-b312-4ab2-b13f-0ee4b6e8bd0f",
+   "metadata": {},
+   "source": [
+    "### Step 5: Display the validation loss\n",
+    "\n",
+    "Now that the results are in, let's visualize the validation loss of the two distilled models using the `tensorboard` library. \n",
+    "> `NOTE:` This notebook demonstrates the use of the teacher finetuning, pruning and the distillation script. These scripts should ideally be run on a multi-node cluster with a larger `GLOBAL_BATCH_SIZE` and `STEPS` to see improvement in the validation loss."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b5822d62-8131-4046-8c22-0bf0fce81df7",
+   "metadata": {},
+   "source": [
+    "#### Validation Loss using depth-pruned model as student in distillation script\n",
+    "Here is an image of the validation loss over 30 steps of running the training step in the distillation script when we distill the knowledge from the finetuned teacher model to the depth-pruned student."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0a665fe1-df45-4126-8694-f182af113133",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%load_ext tensorboard\n",
+    "%tensorboard --logdir \"distill_trainings/megatron_llama_distill_depth_pruned_student/\" --port=6007"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "db6fcf26-8ae8-40e1-875a-0a10bf85be81",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<h5>Validation Loss over 30 Training Steps with Depth-Pruned model as Student</h5>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<img src=\"https://github.com/NVIDIA/NeMo/releases/download/r2.0.0rc1/val_loss_depth_pruned_student_distillation.png\" width=\"400\"/>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from IPython.display import Image, display, HTML\n",
+    "title = \"Validation Loss over 30 Training Steps with Depth-Pruned model as Student\"\n",
+    "display(HTML(f\"<h5>{title}</h5>\"))\n",
+    "display(Image(url=\"https://github.com/NVIDIA/NeMo/releases/download/r2.0.0rc1/val_loss_depth_pruned_student_distillation.png\", width=400))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f10041ae-6533-47de-9f76-f97d4469c27a",
+   "metadata": {},
+   "source": [
+    "#### Validation Loss using width-pruned model as student in distillation script\n",
```

Comment:
fix capitalization

Validation Loss Using Width-Pruned Model as Student in Distillation Script\n",